imgur-screenshot.sh

```diff
@@ -388,6 +445,22 @@ function handle_album_creation_error() {
   fi
 }
 
+function delete_from_imgur() {
```

Comment:
See https://github.com/jomo/imgur-screenshot/pull/111#issuecomment-335328059. I'd be happy to link to this as a separate script in the README / wiki though.

Reply:
I'd be happy to move that into a separate script.  Is it acceptable to leave it in the .desktop icon template as an action?

Reply:
No, users could just edit their .desktop file if they wand and have the script installed.